tools/README.md

```diff
@@ -45,39 +45,34 @@ subcommand help:
 ./maptools.py <subcommand> --help
 ```
 ### Examples
-build all (default):
+build map (default):
 ```
-./maptools.py build all
-```
-
-build only map modules:
-```
-./maptools.py build map
+./maptools.py build
 ```
 
 build only dep modules(nng, safeclib, dwpal):
 ```
 ./maptools.py build dep -n
 ```
 
-Build all with nng messaging library (needs to have nng installed):
+Build map with nng messaging library (needs to have nng installed):
 ```
-./maptools.py build all -f MSGLIB=nng
+./maptools.py build map -f MSGLIB=nng
 ```
 
 Clean and rebuild controller only:
 ```
-./maptools.py build controller -c clean make
+./maptools.py build map -c clean make
 ```
```

Comment:
Actually, building controller only is no longer supported so this paragraph should just be removed.